### ISSUE TITLE
Close all other sidebars when a new sidebar is opened.

### ISSUE THREAD

**Issue by camelburrito** [ampproject/amphtml#3320]
Although not recommended we could still have more than 1 sidebar on a page. 

The issue here is we could have more than 1 sidebar open at the same time. 

For example - open sidebar 1 and have a button inside sidebar 1 to open sidebar 2. This would open sidebar 2 without closing sidebar 2. 

This needs to be fixed. The fix would be is to close all other sidebars on the page when any sidebar is opened.

**Comment by ShadOoW** [comment]
@camelburrito 

```
Although not recommended we could still have more than 1 sidebar on a page.
```

Will the validation error still fire if 2 sidebar are added?

**Comment by ericlindley-g** [comment]
Just tested this, and I'm seeing a validation error for 2 sidebars.

Related: https://github.com/ampproject/amphtml/issues/6104

**Comment by mmicael1** [comment]
Any progress ?

**Comment by camelburrito** [comment]
@cvializ - this might be a good starter bug , let me know if you want to work on it.

**Comment by ericlindley-g** [comment]
I'm on the fence if this is the right way to solve potential issues, as there may be use cases where users want to have two sidebars open at the same time. Adding to the "New FRs" milestone to think through.

**Comment by camelburrito** [comment]
@ericlindley-g The sidebar was built to show one sidebar at a time , why else would we have a mask. We could re-spec it completely on the next version(which will happen in june i guess ). But this version should only display one sidebar at a time

**Comment by aghassemi** [comment]
Removing `Great First Issues` since there is uncertainty here based on previous comments.

**Comment by Lillian-wang** [comment]
@camelburrito  and @aghassemi, based on your current conversation, should I hold off on this bug until you decide whether more than one sidebar open at a time is acceptable?

**Comment by aghassemi** [comment]
@Lillian-wang I would say so. Considering keeping two sidebars open might be a feature to support and also considering there is a easy work-around for this (e.g. Sidebar1 can call `close` on itself before calling `open` on Siebar2) we may not need this feature.

**Comment by camelburrito** [comment]
I think we need the feature , ill talk to our product team and update here

CC @ericlindley-g

**Comment by aghassemi** [comment]
@camelburrito why not leave it up to the developers to decide if they want multiple open-sidebars or not?

If they want sidebar1 to close when sidebar2 is opened, it's a simple `on="tap:sidebar1.close, sidebar2.open`

Opening sidebar2 from sidebar1 without closing sidebar1 can enable interesting use-cases as well For example it allows "sheet" like hierarchal navigation where a link in sidebar1 opens sidebar2 (on top of sidebar1) and sidebar2 can contain sub-links related to link1 and a `back` button that closes it.

**Comment by nainar** [comment]
This is fixed by: #13364. 

Please refer to `amp-accordion.md` under section `expand-single-section`
